python/pdstools/adm/ADMDatamart.py

```diff
@@ -928,7 +931,8 @@ def pivot_df(self, df: pl.LazyFrame, by="Name", allow_collect=True) -> pl.DataFr
             df = (
                 df.collect()
                 .pivot(index=by, columns="PredictorName", values="PerformanceBin")
-                .fill_null(0.5)
+                .fill_null(50)
+                .fill_nan(50)
```

Comment:
Started filling with 0.5
pivot_df function was used in PlotPredictorPerformanceHeatmap. Now we update performance to 0-100 scale in that function.
But slicing the dataframe with .head() instead of [: , :] usage was not possible since the top_n predictors are in columns instead of rows after pivoting them. Maybe we can integrate top_n inside pivot_df to achieve that.

[See commit](https://github.com/yusufuyanik1/pega-datascientist-tools/commit/e64af014c836da72859687f329d4ccc6d27606a6)

Reply:
I'm fine having top_n inside of pivot_df. Otherwise what you could do is something like `df.select(df.columns[:n])`. That would only work in eager mode, but right after a pivot it'll always be eager.